wunderio/next4drupal-project
# NEX-55: Add user registration

About: https://wunder.atlassian.net/browse/NEX-55 

*Changes proposed in this PR:*
* allow users to get create accounts in drupal
* add frontend form and api route to allow users to register
* changes in the UI (user menu now unified)
* automatically assign the `frontend_login` role to users

*How to test:*
Locally: 
1. run the megacommand
2. visit the frontend at https://frontend.lndo.site/
3. use the "Log-in / Create account" menu, select register
4. input username and email, submit the form, you should see a success message
5. visit http://mail.lndo.site/ , you should see the email that the site would have sent to the user
6. copypaste the link and visit it in an incognito browser
7. set a password
8. you should now be able to log in from the frontend using your username and password.
